.github/workflows/release.yml

```diff
@@ -249,3 +249,53 @@ jobs:
         env:
           GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           TAG: ${{ steps.create-release.outputs.computed-prefix }}${{ steps.create-release.outputs.version }}
+
+  check-plugin-version:
+    name: Check if plugin version exist in the registry and upload if not
```

Comment:
Should probably add `needs: [ create-release ]` so it only triggers on successful release.